cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -530,6 +534,19 @@ function(px4_os_add_flags)
 	set(added_link_dirs
 		${nuttx_export_dir}/libs
 		)
+
+	if (${config_nuttx_hw} STREQUAL "m4")
+		set(APPEND added_link_dirs
+			/home/miguel/gcc-arm-none-eabi-5_4-2016q2/lib/gcc/arm-none-eabi/5.4.1/armv7e-m/fpu/
```

Comment:
I'm not sure what the best way to get this path using CMake so I'm open to any suggestions.